docs/update_and_migration/from_2.5/update_from_2.5.md

```diff
@@ -1,14 +1,16 @@
 ---
 description: Update your installation to the latest v3.3 version from v2.5.
 target_version: '3.2'
-latest_tag: '3.3.24'
+latest_tag: '3.3.36'
 ---
 
 # From 2.5
 
 This update procedure applies if you are using v2.5.
 
-Go through the following steps to update to the latest v3.3 LTS (v[[= latest_tag =]]).
+Go through the following steps to update to the v3.3 LTS (v[[= latest_tag =]]).
```

Comment:
Instead of using a local variable, we can use the global one:
```suggestion
Go through the following steps to update to the v3.3 LTS (v[[= latest_tag_3_3 =]]).
```